compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/input/pointer/PointerInputEvent.desktop.kt

```diff
@@ -18,7 +18,7 @@ package androidx.compose.ui.input.pointer
 
 import java.awt.event.MouseEvent
 
-internal actual class PointerInputEvent(
+internal actual data class PointerInputEvent(
```

Comment:
IIRC we cannot upstream code with data classes, so let’s avoid that.

Reply:
Guess you'll need toString here.

Reply:
it is internal. internal data classes are allowed.

it is public API data classes that not allowed, because they are not backward compatible.

Reply:
> Guess you'll need toString here.

I use `copy`